bin/utils.js

```diff
@@ -33,18 +33,33 @@ if (typeof persistenceDir === 'string') {
   const LeveldbPersistence = require('y-leveldb').LeveldbPersistence
   const ldb = new LeveldbPersistence(persistenceDir)
   persistence = {
-    bindState: async (docName, ydoc) => {
+    // Sync in-memory and ldb state
+    mutualSync: async (docName, ydoc) => {
       const persistedYdoc = await ldb.getYDoc(docName)
       const newUpdates = Y.encodeStateAsUpdate(ydoc)
       ldb.storeUpdate(docName, newUpdates)
       Y.applyUpdate(ydoc, Y.encodeStateAsUpdate(persistedYdoc))
-      ydoc.on('update', update => {
+    },
+
+    bindState: async (docName, ydoc) => {
+      await persistence.mutualSync(docName, ydoc)
+      ydoc.on('update', (update) => {
         ldb.storeUpdate(docName, update)
       })
     },
-    writeState: async (docName, ydoc) => {}
+
+    writeState: async (docName, ydoc) => {},
+
+    // Gets a Y.Doc by name, whether in memory or on disk
+    getYDoc: async (docName) => {
```

Comment:
I think that findOrCreateDoc should be preferred as it will also work when another persistence layer is defined.p

The idea of the persistence object is that it only has two properties: `bindState` and `writeState`. 

Should we remove `mutualSync` and `getYDoc` or outsource them to a different function?